README.md

```diff
@@ -31,3 +61,34 @@ You can check if others are experiencing similar issues [here](https://github.co
 | `showCollections`  | `Boolean`         | Show or not the collections badges            |
 | `allowSwap`        | `Boolean`         | Allow or not the item swap                    |
 | `allowRemoval`     | `Boolean`         | Allow or not the item removal                 |
+
+### Styles API
+This app has CSS customization through CSS Modules. CSS Modules is a CSS file in which all class names and animation names are scoped locally by default. You can read more about CSS Modules here .
+
+We use it css-loader to generate a CSS token on a HTML element. For example, the builder generate a CSS token based on app vendor, name and major version. Like container token declared in product-kit, generate the classname vtex.product-kit-1-x-container.
+
+Below, we describe the tokens, their explanation and the component where it is located.
+
+
+| Token name         | Component          | Description                                            |
+| ------------------ | ----------         |------------------------------------------------------- |
+| `listContainer`        | [index](https://github.com/vtex-apps/product-kit/blob/master/react/index.js)           | The main section of the product kit                         |
```

Comment:
I got confused with the descriptions of both `listContainer` and `container`. Could you describe it better?

Reply:
Basically it is something like this: `listContainer` is a wrapper that envolves the `container`. It defines the main paddings and margins. The `container` uses the util area of the products.

`listContainer`
![captura de tela de 2019-02-12 13-48-05](https://user-images.githubusercontent.com/9946330/52652915-b3ba1180-2ecd-11e9-8665-300a4e410a14.png)

`container`
![captura de tela de 2019-02-12 13-49-02](https://user-images.githubusercontent.com/9946330/52652934-bddc1000-2ecd-11e9-8db9-bfa6dad3de56.png)